docs/pages/mintlist/[mintlistAddress].tsx

```diff
@@ -244,5 +279,274 @@ function useMintlist({
+                transactionBase64: GTransaction.toBuffer({
+                  gtransaction: tx,
+                }).toString("base64"),
+                network: network,
+              });
+              resetForm({ values: initialValues });
+            } catch (err) {
+              console.error(err);
+            }
+
+            setSubmitting(false);
+          }}
+        >
+          {({ values, isValid }) => (
+            <Form>
+              <div className="grid">
+                <FieldArray name="nfts">
+                  {({ insert }) => (
+                    <>
+                      {values.nfts.map((_, index) => (
+                        <div key={index}>
+                          <div className="mb-4">
+                            <LuxInputField
+                              placeholder="Name"
+                              name={`nfts.${index}.name`}
+                              required
+                            />
+                          </div>
+
+                          <ImageDropZone
+                            label="NFT Image"
+                            fieldName={`nfts.${index}.image`}
+                          />
+                        </div>
+                      ))}
+                      {values.nfts.length <= MAX_NFTS_PER_BATCH &&
+                        values.nfts.length < availableToUpload && (
+                          <button
+                            type="button"
+                            className="add-nft-button animated"
+                            onClick={() =>
+                              insert(values.nfts.length, {
+                                name: "",
+                                image: "",
+                              })
+                            }
+                          >
+                            <PlusIcon
+                              style={{
+                                width: "2rem",
+                                height: "2rem",
+                              }}
+                            />
+                          </button>
+                        )}
+                    </>
+                  )}
+                </FieldArray>
+              </div>
+              <div className="mt-4 flex-center spread">
+                <LuxSubmitButton
+                  label={`Upload ${values.nfts.length} NFT${
+                    values.nfts.length !== 1 ? "s" : ""
+                  }`}
+                  rounded
+                  color="brand"
+                  disabled={!(isValid && values.nfts.every((nft) => nft.image))}
+                />
+                <LuxButton
+                  label="Disconnect Wallet"
+                  onClick={onSignOut}
+                  color="secondary"
+                  size="small"
+                  variant="link"
+                />
+              </div>
+            </Form>
+          )}
+        </Formik>
+      </InteractiveWell>
+      <style jsx>{`
+        .grid {
```

Comment:
I think we should try to stay away from CSS grid. Whenever anyone has tried it in the past, it's seemed to be more trouble than it's worth. (But maybe it's fine here, just a note for the future.)